core/src/main/scala/org/apache/spark/executor/CoarseGrainedExecutorBackend.scala

```diff
@@ -182,6 +195,8 @@ private[spark] object CoarseGrainedExecutorBackend extends Logging {
       workerUrl.foreach { url =>
         env.rpcEnv.setupEndpoint("WorkerWatcher", new WorkerWatcher(env.rpcEnv, url))
       }
+      env.actorSystem.awaitTermination()
```

Comment:
I don't think we need both awaitTerminations here.  The rpcEnv would cover the actor one so could just do the stop after that.

From SparkEnv:
    val rpcEnv = RpcEnv.create(actorSystemName, hostname, port, conf, securityManager)
    val actorSystem = rpcEnv.asInstanceOf[AkkaRpcEnv].actorSystem